Source/WebKit/UIProcess/BrowsingContextGroup.cpp

```diff
@@ -58,7 +58,7 @@ void BrowsingContextGroup::sharedProcessForSite(WebsiteDataStore& websiteDataSto
         if (websitePolicies && !websitePolicies->allowSharedProcess())
             return completionHandler(nullptr);
     }
-    websiteDataStore.fetchDomainsWithUserInteraction([protectedThis = Ref { *this }, websiteDataStore = Ref { websiteDataStore }, preferences = Ref { preferences }, site = Site { site },
+    websiteDataStore.fetchDomainsWithUserInteraction([protectedThis = Ref { *this }, websiteDataStore = Ref { websiteDataStore }, preferences = Ref { preferences }, site = Site { site }, mainFrameSite = Site { mainFrameSite },
```

Comment:
I've started to take long capture lists like this and put them each on their own line, like continueWithProcessForNavigation

Reply:
Sure, we can do that.